code/project.clj

```diff
@@ -35,6 +35,7 @@
    [buddy/buddy-hashers]
    [buddy/buddy-sign]
    [cc.qbits/spandex :exclusions [org.clojure/clojure]]
+   [camel-snake-kebab "0.4.3"]
```

Comment:
I added this library to convert strings to Camel case for the OpenAPI spec, is that ok ?
I noticed that many libraries have their version in the parent project: should I move the version there or leave it here, what is the rational ?

Reply:
You can leave it here. 
The rule is if it's used in different project we should have it in parent project. If not it's optional.